docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
     volumes:
     - ./nginx.conf:/etc/nginx/nginx.conf
     - ./uci-web-channel.conf:/etc/nginx/sites-enabled/default
-    - ./uci-web-channel/build:/usr/site
+    - ./data/uci-web-channel/build:/usr/site
```

Comment:
This one is a build folder, so should it be mounted in `data`? A bit unsure about this, because it doesn't belong the kind of data that we are mounting in the `./data`.

Reply:
No it will not be mounted. Also, web channel is to be dockerized and fixed, I think. I will add a ticket.

Reply:
Understood! Will remove it.